framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -1324,6 +1326,58 @@ private void addAnnotationToStore(CFStore store, AnnotationMirror anno, Node nod
                 transferResult, firstNode, secondNode, firstValue, secondValue, notEqualTo);
     }
 
+    @Override
+    protected void processConditionalPostconditions(
+            MethodInvocationNode n,
+            ExecutableElement methodElement,
+            Tree tree,
+            CFStore thenStore,
+            CFStore elseStore) {
+        // For String.startsWith(String) and String.endsWith(String), refine the minimum length
+        // of the receiver to the minimum length of the argument.
+        // Basically, MinLen annotations are transferred from the argument to the receiver.
+        // However, a polymorphic qualifier cannot be used to express this, because
```

Comment:
Actually, a polymorphic qualifier would not work because it does not refine the type of receiver expressions.  ( You should just delete this comment, but keep the first two lines.)